packages/client/src/index.test.ts

```diff
@@ -45,3 +45,80 @@ describe('init client', () => {
     expect(codeVerifier.length).toBeGreaterThan(10);
   });
 });
+
+const access_token = 'TvuiR2kU_38PtQcZJM8NFF8I4-0lWnr8C5eWsTmCOON';
+const refresh_token = 'icveztdOZ879up1vSsKMFjLzscKBJ44kIoXMKlWeunN';
+const id_token =
+  'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkNza2w2SDRGR3NpLXE0QkVPT1BQOWJlbHNoRGFHZjd3RXViVU5KQllwQmsifQ.eyJzdWIiOiJ3anNadVc4VWpQd2ciLCJhdF9oYXNoIjoiUkhQejU1YnlHcS1wODFoZnpHVllmQSIsImF1ZCI6ImZvbyIsImV4cCI6MTYzMjgxMjIyNSwiaWF0IjoxNjMyODA4NjI1LCJpc3MiOiJodHRwczovL2xvZ3RvLmRldi9vaWRjIn0.RVAHwv6Zpqz-QuUFLsyr4RomNA7OdeyIyIlXPsnUlg0TG8Vfp2EiR2LMaVFYWQVF-ZlUohZYIoBh6WDi_ZosBbWTOA9J15Jxfyh--0Lv_8F_rLnHiAjnEzCgkQkjrDdEucZuWXgJyKbmFllEbzT_pjEh9MJcoP1g15_wtTbUAyPNfLq4gCj7atvricbr1Yg_1j1rTHWC_9PyaM6TvpL2riBvVZJOapLkw3ueKvSsTMot5k7lF_7bLIpXMrQkRcUpRG16MxBYyu1RdkDrYBowOUkQ_wL57628wKF4D7f12oLxvznzIOIQpsEgG9fja7p2-o05nw9hb-Qxbczb9J0fBjTo4SPf97QEqKFpDYVwC50Ix7jArIDcX6f5SBMGOlYNLgS97LYZWH8qyDqLGuEiE2Crqajh9CAN95AQ0Fi9GoKjCfLC_wxiJFLA0kI6OO2EpuRE09v3Yca3Mtb6DgsgwlIrgz_SEbwVglGKp1D93SfmKVg8v1nUP79SXA0xMLUNIFw6gycqHuBYVqYx6NX2Z9WsVqx4gT4RBZdRDD4eVMEPYnZqFgInFkeAsriZsnq3Fj5aVcdzNgp0UPZmjzNlHudDM5ABvkyi88ao-St0DyP2IQtIqJ-xQOmfwH1N_zPBPy_KTKItPFyYdjQbfY6OvKNuA7nDuUuCSzG8KpBPPG4';
```

Comment:
same here for random generation. we can construct a valid token in the future when needed.